src/main/java/misc/LoginState.java

```diff
@@ -0,0 +1,40 @@
+package misc;
+
+public class LoginState
+{
+	private static boolean loggedIn = false;
+	private static boolean adminLoggedIn = false;
+	private static String loggedInAccount = null;
+
+	public static void logout()
+	{
+		loggedIn = false;
+		adminLoggedIn = false;
+		loggedInAccount = null;
+	}
+
+	public static void login(Account account)
```

Comment:
The other functions are reasonably obvious, but this one could use some light javadocs